lib/membrane/rtp/parser.ex

```diff
@@ -114,10 +131,15 @@ defmodule Membrane.RTP.Parser do
         {:ok, state}
 
       pad ->
-        {{:ok, event: {pad, event}}, state}
+        {{:ok, event: {pad, event}, redemand: :output}, state}
```

Comment:
I believe we don't need to redemand here

Reply:
👍🏻 makes sense.